deployment/ccip/changeset/cs_deploy_chain.go

```diff
@@ -119,8 +119,14 @@ func deployChainContractsForChains(
 		if !ok {
 			return fmt.Errorf("chain %d not found", chainSel)
 		}
-		if existingState.Chains[chainSel].LinkToken == nil || existingState.Chains[chainSel].Weth9 == nil {
-			return fmt.Errorf("fee tokens not found for chain %d", chainSel)
+
+		staticLinkExists := existingState.Chains[chainSel].StaticLinkToken != nil
+		linkExists := existingState.Chains[chainSel].LinkToken != nil
+		weth9Exists := existingState.Chains[chainSel].Weth9 != nil
+		feeTokensAreValid := weth9Exists && (linkExists || staticLinkExists)
+
+		if !feeTokensAreValid {
+			return fmt.Errorf("fee tokens not valid for chain %d", chainSel)
```

Comment:
Lets add some more info to the error to aid in any debugging in the future

```suggestion
			return fmt.Errorf("fee tokens not valid for chain %d, staticLinkExists: %t, linkExists: %t, weth9Exists: %t", chainSel, staticLinkExists, linkExists, weth9Exists)
```